### PR TITLE
Update Maintainers and maintenance policy

### PR DESCRIPTION
**Pull request by chefsalim**
Long overdue cleanup, clarification and sane updates to the maintainers list and policy for Builder. 

Speak now or forever hold your peace.

Signed-off-by: Salim Alam <salam@chef.io>

tenor-10492412

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: maintenance-policy.md [hunk 1]
```diff
@@ -36,14 +36,11 @@ This file is the canonical source for how the Habitat project is maintained.
 ## Maintainer
 
 * Each component may have multiple Maintainers
-* Handles contributions on GitHub - first response to a PR within 48 hours
+* Handles contributions on GitHub and reviews/responds to PRs
```
Comment (baumanj) [inline maintenance-policy.md]: Do we have an issue to make a new SLA for responding to PRs?

File: maintenance-policy.md [hunk 2]
```diff
@@ -36,14 +36,11 @@ This file is the canonical source for how the Habitat project is maintained.
...
 * Is available on [Slack](http://habitat-sh.slack.com/)
-* Is available to answer mailing list questions within 48 hours
-* Weekends and local holidays in the Maintainer’s jurisdiction are not counted
-  for timeliness requirements. Absences for reasonable causes such as vacations,
-  illness, etc. are also acceptable; Maintainers should notice of absences via
-  slack list whenever possible.
-* Committed to 100% tests passing for your component
+* Is available on the Habitat Forum to answer questions
```
Comment (baumanj) [inline maintenance-policy.md]: I thought we were moving away from any commitment for maintainers to be present in Slack and instead focusing on the forums

File: maintenance-policy.md (hunk 1 above)
Comment (chefsalim) [reply maintenance-policy.md]: This is proposing a change for Builder only - I'm happy to open an issue if that will help spur more discussion.  Builder has very few open PRs that are outstanding and I don't anticipate seeing the kinds of backlogs as the Hab repo. However given the reality of manpower commitment I'm also not feeling that we will commit a 48 hr turnaround by maintainers.

File: maintenance-policy.md (hunk 2 above)
Comment (chefsalim) [reply maintenance-policy.md]: Good point - I will remove the Slack commitment.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -1,98 +1,73 @@
...
-* [Steven Murawski](https://github.com/smurawski)
-* [Ian Henry](https://github.com/eeyun)
-* [Matt Wrock](https://github.com/mwrock)
-* [Salim Alam](https://github.com/chefsalim)
-* [Nell Shamrell-Harrington](https://github.com/nellshamrell)
-* [Jon Bauman](https://github.com/baumanj)
-
-### Documentation
+* [Matthew Peck](https://github.com/mpeck)
+* [Shadae](https://github.com/shadae)
```
Comment (raskchanky) [inline MAINTAINERS.md]: Is there a reason Shadae doesn't have a last name?

Comment (chefsalim) [reply MAINTAINERS.md]: Will confirm

File: maintenance-policy.md (hunk 1 above)
Comment (baumanj) [reply maintenance-policy.md]: That's reasonable. I think it's fine for builder to have a different SLA; just that all our community projects should have one.

I'm glad to see this getting updated, regardless.

Comment (chefsalim) [reply maintenance-policy.md]: Re: Governance title - I'm happy to change to a less martial wording - perhaps "Sub-Lead" as there can be at most one.

Comment (baumanj) [reply maintenance-policy.md]: Vice lead?
Highlander?

Comment (eeyun) [reply maintenance-policy.md]: FWIW we are going to be suggesting all our projects have SLAs for PRs but that SLA can absolutely be: "best effort" or "SLA doesn't cover a timed response to PRs" so long as its publically codified. I think its fine to say: first response in a reasonable amount of time  ¯\_(ツ)_/¯

Comment (chefsalim) [reply maintenance-policy.md]: ok, will update

**Pull request merged**